rust_src/remacs-sys/lib.rs

```diff
@@ -180,1022 +29,51 @@ pub struct Lisp_Subr {
+pub const INTMASK: EmacsInt = (EMACS_INT_MAX >> (Lisp_Bits::INTTYPEBITS as u32 - 1));
 extern "C" {
-    pub static mut globals: emacs_globals;
-    pub static Qt: Lisp_Object;
-    pub static Qarith_error: Lisp_Object;
-    pub static Qrange_error: Lisp_Object;
-    pub static Qnumber_or_marker_p: Lisp_Object;
-    pub static Qinteger_or_marker_p: Lisp_Object;
-    pub static Qconsp: Lisp_Object;
-    pub static Qnumberp: Lisp_Object;
-    pub static Qintegerp: Lisp_Object;
-    pub static Qfloatp: Lisp_Object;
-    pub static Qstringp: Lisp_Object;
-    pub static Qsymbolp: Lisp_Object;
-    pub static Qlistp: Lisp_Object;
-    pub static Qmarkerp: Lisp_Object;
-    pub static Qwholenump: Lisp_Object;
-    pub static Qvectorp: Lisp_Object;
-    pub static Qsequencep: Lisp_Object;
-    pub static Qcharacterp: Lisp_Object;
-    pub static Qminus: Lisp_Object;
-
-    pub static Qinteger: Lisp_Object;
-    pub static Qsymbol: Lisp_Object;
-    pub static Qstring: Lisp_Object;
-    pub static Qcons: Lisp_Object;
-    pub static Qmarker: Lisp_Object;
-    pub static Qoverlay: Lisp_Object;
-    pub static Qfinalizer: Lisp_Object;
-    pub static Quser_ptr: Lisp_Object;
-    pub static Qfloat: Lisp_Object;
-    pub static Qwindow_configuration: Lisp_Object;
-    pub static Qprocess: Lisp_Object;
-    pub static Qwindow: Lisp_Object;
-    pub static Qcompiled_function: Lisp_Object;
-    pub static Qbuffer: Lisp_Object;
-    pub static Qframe: Lisp_Object;
-    pub static Qvector: Lisp_Object;
-    pub static Qchar_table: Lisp_Object;
-    pub static Qbool_vector: Lisp_Object;
-    pub static Qhash_table: Lisp_Object;
-    pub static Qthread: Lisp_Object;
-    pub static Qmutex: Lisp_Object;
-    pub static Qcondition_variable: Lisp_Object;
-    pub static Qsubr: Lisp_Object;
-    pub static Qfont_spec: Lisp_Object;
-    pub static Qfont_entity: Lisp_Object;
-    pub static Qfont_object: Lisp_Object;
-    pub static lispsym: Lisp_Symbol;
-    pub static Vbuffer_alist: Lisp_Object;
-
-    pub fn Fcons(car: Lisp_Object, cdr: Lisp_Object) -> Lisp_Object;
-    pub fn Fcurrent_buffer() -> Lisp_Object;
-    pub fn Fsignal(error_symbol: Lisp_Object, data: Lisp_Object) -> !;
-
-    pub fn make_float(float_value: c_double) -> Lisp_Object;
-    pub fn make_string(s: *const c_char, length: ptrdiff_t) -> Lisp_Object;
-    pub fn build_string(s: *const c_char) -> Lisp_Object;
-    pub fn make_unibyte_string(s: *const c_char, length: ptrdiff_t) -> Lisp_Object;
-    pub fn make_uninit_string(length: EmacsInt) -> Lisp_Object;
-    pub fn make_uninit_multibyte_string(nchars: EmacsInt, nbytes: EmacsInt) -> Lisp_Object;
-    pub fn string_to_multibyte(string: Lisp_Object) -> Lisp_Object;
-
-    pub fn SYMBOL_NAME(s: Lisp_Object) -> Lisp_Object;
-    pub fn CHECK_IMPURE(obj: Lisp_Object, ptr: *const c_void);
     pub fn internal_equal(
         o1: Lisp_Object,
         o2: Lisp_Object,
         kind: EqualKind,
-        depth: c_int,
+        depth: std::os::raw::c_int,
```

Comment:
Can bindgen use `libc` types instead? (At least on the command line there is an option for that.)